{"pytest_version": "7.4.3", "$report_type": "SessionStart"}
{"nodeid": "tests57_16_1.py::test_matrix_of_single_number_equal_to_sum", "location": ["tests57_16_1.py", 17, "test_matrix_of_single_number_equal_to_sum"], "keywords": {"test_matrix_of_single_number_equal_to_sum": 1, "__wrapped__": 1, "tests57_16_1.py": 1, "Folder_16": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.0001802939805202186, "start": 1700279035.2482398, "stop": 1700279035.2484214, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "58f26cf9aefe4730bff5f5f1d9f3d4e6", "node": "<WorkerController gw0>"}
{"nodeid": "tests57_16_1.py::test_matrix_of_several_numbers", "location": ["tests57_16_1.py", 29, "test_matrix_of_several_numbers"], "keywords": {"test_matrix_of_several_numbers": 1, "__wrapped__": 1, "tests57_16_1.py": 1, "Folder_16": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00014299500617198646, "start": 1700279035.2540343, "stop": 1700279035.2541778, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "58f26cf9aefe4730bff5f5f1d9f3d4e6", "node": "<WorkerController gw1>"}
{"nodeid": "tests57_16_1.py::test_matrix_of_single_number_equal_to_sum", "location": ["tests57_16_1.py", 17, "test_matrix_of_single_number_equal_to_sum"], "keywords": {"test_matrix_of_single_number_equal_to_sum": 1, "__wrapped__": 1, "tests57_16_1.py": 1, "Folder_16": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/usr/lib/python3/dist-packages/numpy/matrixlib/defmatrix.py", "lineno": 193, "message": "IndexError: index 1 is out of bounds for axis 0 with size 1"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_single_number_equal_to_sum():", "        input_matrix = np.matrix([[-157]])", ">       assert submatrix_with_particular_sum(input_matrix) == [[[-157]]]"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q57/codellama_7b_results_1/Folder_16/tests57_16_1.py", "lineno": 21, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    submatrix.append(matrix[row][col])"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q57/codellama_7b_results_1/Folder_16/generated_answer.py", "lineno": 18, "message": "in submatrix_with_particular_sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def __getitem__(self, index):", "        self._getitem = True", "    ", "        try:", ">           out = N.ndarray.__getitem__(self, index)", "E           IndexError: index 1 is out of bounds for axis 0 with size 1"], "reprfuncargs": {"args": [["self", "matrix([[-157]])"], ["index", "1"]]}, "reprlocals": null, "reprfileloc": {"path": "/usr/lib/python3/dist-packages/numpy/matrixlib/defmatrix.py", "lineno": 193, "message": "IndexError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    @ignore_warnings", "    def test_matrix_of_single_number_equal_to_sum():", "        input_matrix = np.matrix([[-157]])", ">       assert submatrix_with_particular_sum(input_matrix) == [[[-157]]]"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q57/codellama_7b_results_1/Folder_16/tests57_16_1.py", "lineno": 21, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    submatrix.append(matrix[row][col])"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q57/codellama_7b_results_1/Folder_16/generated_answer.py", "lineno": 18, "message": "in submatrix_with_particular_sum"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": ["    def __getitem__(self, index):", "        self._getitem = True", "    ", "        try:", ">           out = N.ndarray.__getitem__(self, index)", "E           IndexError: index 1 is out of bounds for axis 0 with size 1"], "reprfuncargs": {"args": [["self", "matrix([[-157]])"], ["index", "1"]]}, "reprlocals": null, "reprfileloc": {"path": "/usr/lib/python3/dist-packages/numpy/matrixlib/defmatrix.py", "lineno": 193, "message": "IndexError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/usr/lib/python3/dist-packages/numpy/matrixlib/defmatrix.py", "lineno": 193, "message": "IndexError: index 1 is out of bounds for axis 0 with size 1"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.00021464601741172373, "start": 1700279035.2486954, "stop": 1700279035.2489104, "extras": [], "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "58f26cf9aefe4730bff5f5f1d9f3d4e6", "node": "<WorkerController gw0>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
